packages/kbn-apm-synthtrace/src/lib/shared/base_client.ts

```diff
@@ -53,6 +53,17 @@ export class SynthtraceEsClient<TFields extends Fields> {
       )}"`
     );
 
+    const resolvedIndices = this.indices.length
+      ? (
+          await this.client.indices.resolveIndex({
```

Comment:
Thanks for that, I was doing the same thing on another PR that hasn't been merged yet 😆